multi_step_wizard/tests/test_multi_step_wizard.py

```diff
@@ -1,20 +1,24 @@
 # Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo_test_helper import FakeModelLoader
 
-from odoo.tests.common import TransactionCase
+from odoo.tests import common
 
 
-class TestMultiStepWizard(TransactionCase):
-    def setUp(self):
-        super().setUp()
-        self.MultiStepWizard = self.env["multi.step.wizard.test"]
+class CommonTierValidation(common.SavepointCase):
```

Comment:
`TestMultiStepWizard` and replace all calls to super w/ `super()`, no need for the class name.